python/ray/dashboard/modules/metrics/dashboards/data_dashboard_panels.py

```diff
@@ -143,147 +143,145 @@
     # Ray Data Metrics (Inputs)
     Panel(
         id=17,
-        title="Input Blocks Received by Operator",
-        description="Number of input blocks received by operator.",
-        unit="blocks",
+        title="Input Blocks Received by Operator / Second",
+        description="Number of input blocks received by operator per second.",
+        unit="blocks/sec",
         targets=[
             Target(
-                expr="sum(ray_data_num_inputs_received{{{global_filters}}}) by (dataset, operator)",
-                legend="Blocks Received: {{dataset}}, {{operator}}",
+                expr="sum(rate(ray_data_num_inputs_received{{{global_filters}}}[30s])) by (dataset, operator)",
+                legend="Blocks Received / Second: {{dataset}}, {{operator}}",
             )
         ],
         fill=0,
         stack=False,
     ),
     Panel(
         id=18,
-        title="Input Blocks Received by Operator",
-        description="Byte size of input blocks received by operator.",
-        unit="bytes",
+        title="Input Blocks Received by Operator / Second",
```

Comment:
this is a typo in the existing code.
```suggestion
        title="Input Bytes Received by Operator / Second",
```